python/tk_desktop2/websockets/encryption_handler.py

```diff
@@ -75,8 +75,8 @@ def decrypt(self, payload):
         :param str payload: String to decrypt.
         :returns: Decrypted string
         """
-        b = bytes(payload)
-        return self._fernet.decrypt(b)
+        b = six.b(payload)
```

Comment:
fernet expect bytes

Reply:
If your payload had unicode characters, would the conversion to latin-1 (what six.b does) really work?